lib/online_order.rb

```diff
@@ -0,0 +1,61 @@
+require "csv"
+require "awesome_print"
+
+module Grocery
+  class OnlineOrder < Order
+    attr_reader :id, :customer, :products, :status
```

Comment:
Since this class inherits from `Order`, you don't need to define an `attr_reader` for `id` and `products` because they inherit them from `Order`.